documentation/asciidoc/computers/configuration/headless.adoc

```diff
@@ -1,6 +1,6 @@
 == Setting up a Headless Raspberry Pi
 
-If you do not use a monitor or keyboard to run your Pi (known as headless), but you still need to do some wireless setup, there is a facility to enable wireless networking and SSH when creating a image.
+If you do not use a monitor or keyboard to run your Raspberry Pi (known as headless), but you still need to do some wireless setup, there is a facility to enable wireless networking and SSH when creating an image.
 
 Once an image is created on an SD card, by inserting it into a card reader on a Linux or Windows machines the xref:configuration.adoc#the-boot-folder[boot folder] can be accessed. Adding certain files to this folder will activate certain setup features on the first boot of the Raspberry Pi.
```

Comment:
@aallan Perhaps it's worth adding some wording here about the built-in support for this in RPi-Imager, now that it's exposed without requiring a special keypress?

Reply:
how about would like to add a content related to the Raspberry Pi Imager to the technical document. and add a separate guideline for the Raspberry Pi Imager?